internal/execute/table/mask.go

```diff
@@ -0,0 +1,97 @@
+package table
+
+import (
+	"github.com/apache/arrow/go/arrow/array"
+	"github.com/influxdata/flux"
+	"github.com/influxdata/flux/execute"
+	"github.com/influxdata/flux/values"
+)
+
+func Mask(tbl flux.Table, columns []string) flux.Table {
```

Comment:
I guess you could have put this in your `contrib` directory. Why are you exporting it here, do you think it will be useful outside of for the `_mask` transformation?

Reply:
If it's going to be outside of `contrib`, I think that it needs tests.

Reply:
Done.